### PR TITLE
Typos fixes

### PR DESCRIPTION
**Pull request by GernotMaier**
Typo fixes in comments found while browsing through the code.

Should not change anything (apart from being with less typos)

### CODE REVIEW

File: simtools/util/names.py [hunk 1]
```diff
@@ -743,7 +743,7 @@ def corsikaOutputFileName(run, primary, arrayName, site, zenith, azimuth, label=
 
     Warning
     -------
-        zst extension is hardcoded here.
+        zst extension is hard coded here.
```
Comment (orelgueta) [inline simtools/util/names.py:746]: One word is actually OK (in particular in this context).

File: simtools/util/names.py [hunk 2]
```diff
@@ -839,7 +839,7 @@ def simtelOutputFileName(run, primary, arrayName, site, zenith, azimuth, label=N
...
```
Comment (orelgueta) [inline simtools/util/names.py:842]: One word is actually OK (in particular in this context).

File: simtools/util/names.py [hunk 3]
```diff
@@ -875,7 +875,7 @@ def simtelHistogramFileName(run, primary, arrayName, site, zenith, azimuth, labe
...
```
Comment (orelgueta) [inline simtools/util/names.py:878]: One word is actually OK (in particular in this context).

File: simtools/util/names.py [hunk 4]
```diff
@@ -911,7 +911,7 @@ def simtelLogFileName(run, primary, arrayName, site, zenith, azimuth, label=None
...
```
Comment (orelgueta) [inline simtools/util/names.py:914]: One word is actually OK (in particular in this context).

**Review by orelgueta — COMMENTED** [review]
Looks good except the change of `hardcoded` to `hard coded` which is unnecessary.

File: simtools/util/names.py (hunk 4 above)
Comment (GernotMaier) [reply simtools/util/names.py]: OK - reversed that. Although technically wrong, one would probably search the code for hardcoded.

**Pull request merged**